frontend/app/settings/page.tsx

```
@@ -28,51 +35,226 @@ const DetailsSection = () => {
+          </MutationStatusButton>
+        </form>
+      </Form>
+    </div>
+  );
+};
+
+const WorkspaceAccessSection = () => {
+  const user = useCurrentUser();
+  const [showLeaveModal, setShowLeaveModal] = useState(false);
+  const currentCompany = user.companies.find((company) => company.id === user.currentCompanyId);
+  console.log("Current Company:", currentCompany);
```

ESLint

[error] Unexpected console statement.

(no-console)